client/src/App.js

```diff
@@ -1,13 +1,193 @@
-import "./App.css";
+
+
+import React, { useState, useEffect, useCallback } from 'react';
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import VideoDisplay from './VideoDisplay';
+import AddNewVideo from './AddNewVideo';
+import VideoDetails from './VideoDetails';
+import VideosData from './VideosData';
+import axios from 'axios';
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Video Recommendation</h1>
-      </header>
-    </div>
-  );
+const [videos, setVideos] = useState(VideosData);
+const [order, setOrder] = useState('asc');
+// const [ascending, setAscending] = useState(false);
+// videosData.sort((a,b) => b.rating - a.rating)
+
+
+
+
+
+
+const fetchVideos= useCallback(async () => {
+try {
+const response = await fetch(`http://127.0.0.1:8000/videos?order=${order}`);
+if (response.ok) {
+const data = await response.json();
+setVideos(data);
+} else {
+console.log('Error fetching videos:', response.statusText);
+}
+} catch (error) {
+console.log('Error fetching videos:', error);
+}
+}, [order]);
+
+
+
+
+useEffect(() => {
+fetchVideos();
+}, [fetchVideos]);
```

Comment:
What does adding `fetchVideos` to this array do? What would be different if you removed it?